### DISCUSSION TITLE
Add macOS and Linux support to documentation

### DISCUSSION THREAD

**Discussion by kimmy-wang** [codemagic-ci-cd/codemagic-docs#997] [Q&A] [answered]
https://docs.codemagic.io/yaml-quick-start/building-a-flutter-app/

**Comment by himesh-cm** [comment, marked as answer]
Thanks for the suggestion @kimmy214. 

We will discuss it internally.
For mac, you can find our sample project [here](https://github.com/codemagic-ci-cd/codemagic-sample-projects/tree/main/flutter/flutter-macos-demo-project)

Let us know if you need any assistance building on Linux or macOS with Flutter.

Thank you.
